### DISCUSSION TITLE
Git command for clean update?

### DISCUSSION THREAD

**Discussion by Svardsten53** [mcguirepr89/BirdNET-Pi#332] [Q&A]
What is the correct git command (or commands) to make a completely clean update? No merging of data, just downloading the latest version of all files.

I'm working on a package which are doing translations and link modifications and need to have new and clean files to work with after each update. I have tried 'git pull' and 'git fetch', but not all files are updated.

**Comment by mcguirepr89** [comment]
You will get the latest version of all files if you perform the following:
```
git checkout -f main
git pull -f
update_birdnet.sh
```

**Reply by mcguirepr89** [reply]
After doing that once, you will should always use "Tools" > "System Controls" > "Update" anytime thereafter

**Reply by Svardsten53** [reply]
Thanks Patrick! I have tried this a couple of times on both my installations and it seems to work fine. This makes things easier.
